efficientdet/keras/util_keras.py

```diff
@@ -180,14 +224,18 @@ def fp16_to_fp32_nested(input_nested):
   return out_tensor_dict
 
 
-def get_batch_norm(bn_class):
-  def _wrapper(*args, **kwargs):
-    if not kwargs.get('name', None):
-      kwargs['name'] = 'tpu_batch_normalization'
-    return bn_class(*args, **kwargs)
-  return _wrapper
-
-if tf.compat.v1.executing_eagerly_outside_functions():
-  utils.BatchNormalization = get_batch_norm(tf.keras.layers.BatchNormalization)
-  utils.SyncBatchNormalization = get_batch_norm(tf.keras.layers.experimental.SyncBatchNormalization)
-  utils.TpuBatchNormalization = get_batch_norm(tf.keras.layers.experimental.SyncBatchNormalization)
\ No newline at end of file
+### The following code breaks COCO training.
```

Comment:
@fsx950223  The following code breaks COCO training.

Reply:
What's problem, bad performance or runtime error.